requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas==1.2.3
+prometheus_api_client==0.4.2
+numpy
+statsmodels==0.12.2
+opni-nats==0.0.0.2
```

Comment:
Currently should be using opni-nats version 0.0.0.4

Reply:
updated to latest version